ci-operator/step-registry/openshift/release/analysis/prpqr-aggregator/OWNERS

```diff
@@ -0,0 +1 @@
+../../OWNERS
```

Comment:
If this is something that our tooling will trigger, then we should own it, not release-controller owners.

Reply:
I'll open a follow-up